Mf2/Parser.php

```diff
@@ -238,6 +238,24 @@ function convertTimeFormat($time) {
 	}
 }
 
+/**
+ * Normalize an ordinal date to YYYY-MM-DD
+ * @param string $dtValue
+ * @return string
+ */
+function normalizeOrdinalDate($dtValue) {
```

Comment:
This function isn't properly validating the input, but I see that it is only ever called after the input is guaranteed to be in the format `\d{4}-\d{2}`. Can you add a comment to the function definition that explains that this function can only be called with pre-validated input?